packages/components/pie-spinner/test/visual/pie-spinner.spec.ts

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Are all these empty lines at the start of files because of the generator?

Reply:
yes, didn't touch them :,)